app/services/embedded_content_finder_service.rb

```diff
@@ -24,23 +25,44 @@ def find_content_references(value)
 
   def live_content_ids(content_references)
     found_editions = live_editions(content_references.uniq)
-    not_found_content_ids = content_references.map(&:content_id) - found_editions.map(&:content_id)
+    not_found_content_ids = content_references.map(&:identifier) - found_editions.map(&:content_id)
 
     if not_found_content_ids.any?
       GovukError.notify(CommandError.new(
                           code: 422,
                           message: "Could not find any live editions for embedded content IDs: #{not_found_content_ids.join(', ')}",
                         ))
     end
-    content_references.map(&:content_id) - not_found_content_ids
+    content_references.map(&:identifier) - not_found_content_ids
+  end
+
+  def transform_aliases_to_content_ids(content_references)
+    embedded_aliases = content_references.select(&:identifier_is_alias?).map(&:identifier)
+    content_id_aliases = ContentIdAlias.where(name: embedded_aliases).map { |a| [a.name, a.content_id] }.to_h
```

Comment:
I'm struggling a bit to understand the workflow of a ContentIdAlias...
Looking at `put_content`, [they are created when a param is passed in the `payload` ](https://github.com/alphagov/publishing-api/commit/15753423c1f19b279b7bfa450d0b6f24dc08c427)?  So would the `ContentIdAlias` be created/updated when a Block is published?

Reply:
And we create an alias using `friendly_id` here https://github.com/alphagov/whitehall/blob/a73ddeaad983bf3a2ac6282d72950083e420693c/lib/engines/content_block_manager/app/models/content_block_manager/content_block/document.rb#L11